### PR TITLE
copy engine added

### PR DESCRIPTION
**Pull request by leszczyn**

### CODE REVIEW

File: include/ze_info/zenon.hpp [hunk 1]
```diff
@@ -92,6 +92,17 @@ class zenon
     ze_group_count_t group_count = {};
     ze_command_queue_desc_t command_queue_descriptor = {};
     ze_command_queue_handle_t command_queue;
+    
+    ze_command_queue_handle_t input_copy_command_queue;
+    ze_command_list_handle_t input_copy_command_list;
+    ze_command_queue_desc_t input_copy_command_queue_descriptor = {};
+    ze_command_list_desc_t input_copy_command_list_descriptor = {};
+
+    ze_command_queue_handle_t output_copy_command_queue;
```
Comment (tponieck) [inline include/ze_info/zenon.hpp:101]: input copy and output copy could be done with one cmdlist synchronized on events. Consider this as next step

File: src/zenon.cpp [hunk 2]
```diff
@@ -113,10 +113,20 @@ void zenon::init()
...
+                copyOnlyQueueGroupOrdinal = i;
+                break;
+            }
         }
 
-        command_queue_count = cmdqueueGroupProperties[computeQueueGroupOrdinal].numQueues;
-      
+        command_queue_count = cmdqueueGroupProperties[ computeQueueGroupOrdinal ].numQueues;
+
+        //if( !( device_properties.flags & ZE_DEVICE_PROPERTY_FLAG_INTEGRATED ) )
```
Comment (tponieck) [inline src/zenon.cpp:127]: this can lead to run on integra

File: include/ze_info/zenon.hpp (hunk 1 above)
Comment (leszczyn) [reply include/ze_info/zenon.hpp:101]: its in my mind, i think we can add it a bit later , i will add appropietate  task in teams

File: src/zenon.cpp (hunk 2 above)
Comment (leszczyn) [reply src/zenon.cpp:127]: ive added just a comment there.

**Pull request merged**